ppd/ppd-test.c

```diff
@@ -160,20 +160,20 @@ cups_array_t *ppdTest(int ignore,          // Which errors to ignore
   if (q_with_v == 1)
   {
   
-    _ppdArrayAddStrings(output,
-                        "testppdfile: The -q option is incompatible with the -v option.","\0");
+    ppdArrayAdd(output,
+                        "testppdfile: The -q option is incompatible with the -v option.");
```

Comment:
This is the function `ppdTest()` which is not necessarily only be used by the `testppdfile` utility. Therefore please replace the "`testppdfile:`" prefixes in the whole function by "`ppdTest:`".